src/evagram_input/input_tool/input_tool.py

```diff
@@ -15,41 +15,78 @@ def __init__(self, owner: str, experiment: str, eva_directory: str):
         self.eva_directory = eva_directory
         self.owner_id = None
         self.experiment_id = None
+
+        self._status = "OPEN"
```

Comment:
It is referring to the database connection in the workflow session where it would be "open" when it is still currently running and inserting diagnostics into the database. This can be useful to make sure any database cursors or pointers in the session object have been closed to avoid memory leaks.

Reply:
To be completely sure, use `connection.closed`.  I trust psycopg's word more than our own.  The supplementary messages (e.g. "CLOSED WITH SUCCESS") are helpful!  so we can still persist those under a different name, like "status_message".  Then include both status and message in the `repr`